### PR TITLE
add a feedback action to the appbar

### PR DESCRIPTION
**Pull request by devoncarew**
- add a feedback action to the appbar

This restores the feedback action we had in the dart:html version of the app. I'm not sure where the `ReportBugAction` class should go in the codebase.

Also, I tried to make the bullet spacer use the unselected app bar color, but wasn't able to get a non-null color value. I'd be interested in advice about how to calculate the right color...

Screen Shot 2020-03-12 at 2 16 21 PM

### CODE REVIEW

File: packages/devtools_app/lib/src/flutter/app.dart [hunk 1]
```diff
@@ -164,3 +167,40 @@ class _AlternateCheckedModeBanner extends StatelessWidget {
...
+    return InkWell(
+      onTap: () async {
+        // TODO(devoncarew): Support analytics.
+        // ga.select(ga.devToolsMain, ga.feedback);
+
+        const reportIssuesUrl = 'https://github.com/flutter/devtools/issues';
+
+        if (await url_launcher.canLaunch(reportIssuesUrl)) {
+          await url_launcher.launch(reportIssuesUrl);
+        }
```
Comment (jacob314) [inline packages/devtools_app/lib/src/flutter/app.dart]: should we display a toast if we can't launch the url?

File: packages/devtools_app/lib/src/flutter/app.dart [hunk 2]
```diff
@@ -164,3 +167,40 @@ class _AlternateCheckedModeBanner extends StatelessWidget {
     );
   }
 }
+
+class _BulletSpacer extends StatelessWidget {
```
Comment (jacob314) [inline packages/devtools_app/lib/src/flutter/app.dart]: common_widgets.dart could be a good place for this class along with a TODO to use the correct color.

File: packages/devtools_app/lib/src/flutter/app.dart [hunk 3]
```diff
@@ -164,3 +167,40 @@ class _AlternateCheckedModeBanner extends StatelessWidget {
...
+        // ga.select(ga.devToolsMain, ga.feedback);
+
+        const reportIssuesUrl = 'https://github.com/flutter/devtools/issues';
+
+        if (await url_launcher.canLaunch(reportIssuesUrl)) {
+          await url_launcher.launch(reportIssuesUrl);
+        }
+      },
+      child: Container(
+        constraints: const BoxConstraints.tightFor(width: 48.0, height: 48.0),
```
Comment (jacob314) [inline packages/devtools_app/lib/src/flutter/app.dart]: just setting width and height on the container should have the same effect and be a bit clearer.

File: packages/devtools_app/lib/src/flutter/app.dart (hunk 2 above)
Comment (devoncarew) [reply packages/devtools_app/lib/src/flutter/app.dart]: done

File: packages/devtools_app/lib/src/flutter/app.dart (hunk 1 above)
Comment (devoncarew) [reply packages/devtools_app/lib/src/flutter/app.dart]: Will do.

File: packages/devtools_app/lib/src/flutter/app.dart (hunk 3 above)
Comment (devoncarew) [reply packages/devtools_app/lib/src/flutter/app.dart]: updated

**Pull request merged**